src/guides/v2.4/install-gde/prereq/php-settings.md

```diff
@@ -69,9 +76,18 @@ PHP Warning:  date(): It is not safe to rely on the system's timezone settings.
 
    Our detailed recommendations are:
 
-   -  Compiling code or deploying static assets, `756M`
-      -  Installing and updating Magento components from Magento Marketplace, `2G`
-      -  Testing, `~3-4G`
+   -  Compiling code or deploying static assets, `1G`
+   -  Debugging, `2G`
+   -  Testing, `~3-4G`
+
+-  Increase the values for the PHP `realpath_cache_size` and `realpath_cache_ttl` to recommended settings:
+
+   ```conf
+   realpath_cache_size=10M
```

Comment:
IN file magento-app-php-ini.md it says the recommended value is 1M, which one is it?